tests/test_integration.py

```diff
@@ -156,8 +103,67 @@ def test_declined_card_props(self):
             self.assertTrue(e.request_id.startswith('req_'))
 
 
-class InvalidRequestErrorTest(StripeTestCase):
+# TODO: find a way to move all the ones below those to stripe-mock or to stub
```

Comment:
@ob-stripe Flagging this and all the ones below to avoid missing them. stripe-mock seems to be broken (unless I missed something which is likely).

Basically it returns a string for the error that is not a JSON payload and does not set proper headers such as req_XXXX. I filed https://github.com/stripe/stripe-mock/issues/31 though I spent around an hour trying to fix it to no avail and cut my losses.

Reply:
Nice catch! Once your fix in https://github.com/stripe/stripe-mock/pull/32 lands, we'll just need to update `.travis.yml` to pull the updated stripe-mock, and update the `MOCK_MINIMUM_VERSION` in `helper.py` accordingly.